build-tools/src/main/java/org/elasticsearch/gradle/testclusters/RunTask.java

```diff
@@ -65,6 +72,16 @@ public Boolean getDebug() {
         return debug;
     }
 
+    @Input
+    public Boolean getApmServerEnabled() {
+        return apmServerEnabled;
+    }
+
+    @Option(option = "with-apm-server", description = "Run simple logging http server to accept apm requests")
```

Comment:
Should we document this option [here](https://github.com/elastic/elasticsearch/blob/main/TESTING.asciidoc#other-useful-arguments)?

Reply:
good idea. added documentation